stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetResult.kt

```diff
@@ -0,0 +1,21 @@
+package com.stripe.android.paymentsheet
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
```

Comment:
@yuki-stripe is there consistent documentation we should used for Completed, Cancel, and Failed?  This would be used in the javadoc

Reply:
```
/// The result of an attempt to confirm a PaymentIntent or SetupIntent
@frozen public enum PaymentSheetResult {
    /// The customer completed the payment or setup
    /// - Note: The payment may still be processing at this point; don't assume money has successfully moved.
    ///
    /// Your app should transition to a generic receipt view (e.g. a screen that displays "Your order is confirmed!"), and
    /// fulfill the order (e.g. ship the product to the customer) after receiving a successful payment event from Stripe -
    /// see https://stripe.com/docs/payments/handling-payment-events
    case completed

    /// The customer canceled the payment or setup attempt
    case canceled

    /// The attempt failed.
    /// - Parameter error: The error encountered by the customer. You can display its `localizedDescription` to the customer.
    case failed(error: Error)
}
```

Reply:
Updated docs.